middlewares/replace_path.go

```diff
@@ -18,3 +18,8 @@ func (s *ReplacePath) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	r.URL.Path = s.Path
 	s.Handler.ServeHTTP(w, r)
 }
+
+// SetHandler sets handler
+func (s *ReplacePath) SetHandler(Handler http.Handler) {
```

Comment:
To be honest, I'm not sure if this is being used or not, just saw it missing from this rule and present in `addPrefix.go` and `stripPrefix.go`, so added it here for consistency.  If you think it's not needed, I'll be glad to remove it. (love deleting code!!!)